src/Http/Controllers/Views/KpiController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Biigle\Modules\Kpis\Http\Controllers\Views;
+
+use Carbon\Carbon;
+use Biigle\Modules\Kpis\User;
+use Biigle\Modules\Kpis\Storage;
+use Biigle\Modules\Kpis\Requests;
+use Biigle\Http\Controllers\Views\Controller;
+
+class KpiController extends Controller
+{
+    /**
+     * Shows the kpis page.
+     *
+     * @param int $idx month index
+     * @return \Illuminate\Http\Response
+     */
+    public function show($idx = 6)
+    {
+        // If index is not in range, set it on index of last month
+        $idx = $idx == 0 || $idx > 6 ? 6 : $idx;
```

Comment:
What about `$idx = -1`? :wink:
```suggestion
        $idx = min(6, max(0, $idx));
```

Reply:
This case is catched [here](https://github.com/biigle/kpis/blob/564c45a02a0a31571fe27b149600fd1d84ff2315/src/Http/web.php#L9).

Reply:
Ok. Then there will be a 404 response for -1 and you should probably also return a 404 for `$idx > 6`.